examples/kubernetes-in-docker/utils.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+check_env_var() {
+  if [[ -z "${!1+x}" ]]; then
+    echo "You must set $1 before running these scripts."
```

Comment:
```suggestion
    echo "$1 must be exported before running these scripts."
```
This check will fail if the variables are only set, but not exported

Reply:
Good catch, thanks!